combustion-android-ble/src/main/java/inc/combustion/framework/ble/uart/meatnet/NodeResponse.kt

```diff
@@ -80,7 +80,7 @@ internal open class NodeResponse(
 
             val rawMessageType = typeRaw and RESPONSE_TYPE_FLAG.inv()
             val messageType = NodeMessageType.fromUByte(rawMessageType)
-                ?: rawMessageType
+                ?: null
```

Comment:
i think elvis operator can be removed if the alternative is null anyway

Reply:
Removed.